Scripts/runTest.sh

```diff
@@ -1,11 +1,5 @@
 #!/bin/bash
 
-xcodebuild -project GrowthBook-IOS.xcodeproj \
-   -scheme GrowthBook \
-   -sdk iphonesimulator \
-   -destination 'platform=iOS Simulator,name=iPhone 13,OS=16.2'
```

Comment:
This old phone ndeinstation wasn't available on Xcode 15.2 .. that was the issue. so instead the CLI tried to use macOS as destination which was a problem